src/hex-code.ts

```
@@ -0,0 +1,48 @@
+// with help of Claude
+
+export function binaryToHex(binary: string): string {
+  // validation input
+  if (!/^[01]+$/.test(binary) || binary.length % 4 !== 0) {
+    throw new Error("Invalid binary string");
+  }
+
+  let hex = "";
+  for (let i = 0; i < binary.length; i += 4) {
+    const chunk = binary.slice(i, i + 4);
+    const decimal = parseInt(chunk, 2);
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)